ComponentKit/LayoutComponents/CKStackPositionedLayout.mm

```diff
@@ -59,9 +71,12 @@ static CKStackPositionedLayout stackedLayout(const CKStackLayoutComponentStyle &
       l.layout,
     };
     p = p + directionPoint(style.direction, stackDimension(style.direction, l.layout.size) + l.child.spacingAfter, 0);
+    if(crossDimension(style.direction, l.layout.size) > maxLineHieght) {
+        maxLineHieght = crossDimension(style.direction, l.layout.size);
+    }
     return c;
   });
-  return {stackedChildren, crossSize};
+  return {stackedChildren, style.flexWrap == CKStackLayoutWrapFlexWrap ? (maxLineHieght + crossPosition) : crossSize};
```

Comment:
You are dealing with a positioned layout here, but the sizes of the items has already been determined by the unpositioned layout algorithm. Don't you need to update that algorithm as well? Maybe I'm missing something though... it's been a long time since I worked on this code.